templates/zerver/help/glossary.md

```diff
@@ -0,0 +1,55 @@
+# Glossary
+Several terms will be used in Zulip development conversations. Here are some common terms that you should know.
```

Comment:
I think the target audience of this guide is the general users so we should not use 'development conversations'.